### ISSUE TITLE
nba example: pc.select cutoff (0.1.32)

### ISSUE THREAD

**Issue by milochen0418** [pynecone-io/pynecone-examples#83]
```
My Testing Environment
        OS Darwin 22.4.0
        Pynecone 0.1.32
        Python 3.11.3
        Node v16.8.0
        Bun 0.5.9
```
**0.1.31**

<img width="1105" alt="Screenshot 2023-05-29 at 4 17 36 PM" src="https://github.com/pynecone-io/pynecone-examples/assets/12568287/476c03f8-b0fd-4396-99dc-f45a3bc94b87">

<img width="858" alt="Screenshot 2023-05-29 at 4 17 41 PM" src="https://github.com/pynecone-io/pynecone-examples/assets/12568287/a6bbd503-73a4-45fd-b0ef-cf9385715325">

**0.1.32**
<img width="739" alt="Screenshot 2023-05-29 at 4 14 09 PM" src="https://github.com/pynecone-io/pynecone-examples/assets/12568287/4e4e8de8-c4e4-48ac-bfe2-fcc2e8da8275">

**Comment by milochen0418** [comment]
Use pc.container may solve this issue because pc.vstack does not work well for pc.select in pc 0.1.32

**Comment by milochen0418** [comment]
Solved in pc 0.1.33
